### DISCUSSION TITLE
Multi-threaded find_min_global() not using 100% of all cores.

### DISCUSSION THREAD

**Discussion by pfeatherstone** [davisking/dlib#3031] [Q&A] [answered]
I'm using `find_min_global()` with and without a thread pool. When running it without it's using 100% of a single core. When running it with a thread pool of size 8 (my CPU has 8 physical threads), it's using about 11% per core. Anybody else seen this?

**Comment by davisking** [comment, marked as answer]
The threading is only used to call the objective function.  All the mechanism of the solver is single threaded.  So it's only going to help if your objective function is slow.
